tuneme/context_processors.py

```diff
@@ -0,0 +1,11 @@
+from tuneme.forms import LoginForm, RegistrationForm, EditProfileForm
+from tuneme.forms import ProfilePasswordChangeForm
```

Comment:
You can replace these forms with forms from `molo.profiles.forms` and delete `tuneme/forms.py`